documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,575 @@
+merely a guideline that we expect contributors to adhere to. It cannot cover
+every situation and undoubtedly there are exceptional situations where following
+the process to the letter would be antithetical to the goals of the project and
+even of the process itself.
+
+If you have ideas for improving the Openverse decision-making process, please
+share them. Any contributor may provide feedback on this process.
+
+## Terms
+
+- "Author": The person making the proposal.
+- "Participants": All participants of the decision-making process, including the
+  author, the requested participants, and ad-hoc participants.
+- "Reactions": A broad class of feedback that can include personal preferences
+  as well as general doubts about the validity or safety of the solutions
+  described in the proposal.
+- "Question": A request for clarification, explanation, or additional
+  information about an aspect of the proposal.
+- "Objection": An explained uncertainty about the appropriateness, efficacy, or
+  safety of a solution
+- "Paramount objection": An objection outlining an issue that would cause harm
+  to the project or its contributors; especially issues that could cause harm
+  and are difficult to reverse and not strictly necessary to accomplish the
+  goals of the project
+- "Approved proposal": A proposal that has undergone the formal decision-making
+  process and has been accepted as a valid solution by the participants of the
+  discussion.
+- "Process proposal": A proposal to change processes followed by the
+  contributors in the course of stewarding the project towards success.
+- "Purpose": The motivation for the proposal. This is typically a statement
+  describing the problem the proposal aims to solve. In the original Sociocratic
+  model this is called the "driver".
+
+## Process summary
+
+The process breaks proposal evaluation and decision-making into a series of
+rounds. Each round has expectations about what kinds of feedback are
+appropriate. These steps are ordered and must be followed in the order they are
+presented here.
+
+Full descriptions for each of these rounds are below. This section is a summary
+meant to give an overview for people new to or refreshing themselves on the
+process.
+
+> **Note** The "goal" column is a very brief summary and is not intended to
+> encapsulate the full scope of the round. Please refer to the
+> [round descriptions below](#round-descriptions) for more details and examples.
+
+| round                                                 | Suggested span      | Goal                                                                                      |
+| ----------------------------------------------------- | ------------------- | ----------------------------------------------------------------------------------------- |
+| [Question round](#question-round)                     | 3 days              | Clarify the proposal; share initial reactions                                             |
```

Comment:
I don't have a strong preference for a specific number of days. I err on the shorter side personally because of the following reasons:

1. Decision-making and discussion are part of our regular work. The only PR review priority that is 5 days are low priority PRs. 3 days sits between high and medium in this scale. Considering many discussions block further work, 3 days seems like an appropriate amount of time.
2. This is a suggested span and can be expanded by the author if, for example, the proposal is very long, or if the participants request it. It's also not a "hard" deadline in that (as specified in the round description) if there are still important outstanding questions, the round should continue.

Anything less than a week still brings the sense of urgency and expediency that I hope this will help us capture in our decision-making. 5 days sounds fine to me in that sense, but I still think we can consider it in light of the ideas I shared above.

I also think this proposal qualifies for "very long" and I anticipate it going over 3 days in the initial round. On the other hand, the #873 review did not take longer than 5 days total once the requested reviewers took a look at it. If we go for a longer period of time, I'd hope we can more proactively use the shortcutting signal instead of letting a round expire on its own.

Reply:
> I'd hope we can more proactively use the shortcutting signal instead of letting a round expire on its own.

💯 I love all the levers we have here for shortening/lengthening the spans as necessary -- this process is going to be useful for such a wide range of discussions that there's no way to make a default that will make sense in _all_ cases.

The only pro I see for 5 is that it makes overlap with a community meeting part of the _default_, rather than something an author might have to go out of their way to do. But that's such a specific detail, I'm 100% down to just wait and see what the time spans end up looking like once we roll out the process.